src/server/chromium.ts

```diff
@@ -63,6 +64,20 @@ export class Chromium extends BrowserTypeBase {
     return CRBrowser.connect(transport, options, devtools);
   }
 
+  _rewriteStartupError(error: Error): Error {
+    if (!error.message.includes('--no-sandbox'))
```

Comment:
Hmm... If I pass '--no-sandbox' in my args, will this work?

Reply:
Nice catch! Updated with a white-list from [chromium source](https://source.chromium.org/chromium/chromium/src/+/master:content/browser/zygote_host/zygote_host_impl_linux.cc;l=88?q=%22If%20you%20want%20to%20live%20dangerously%20and%20need%20an%20immediate%20workaround%22&ss=chromium&originalUrl=https:%2F%2Fcs.chromium.org%2F)